boundary/collect.py

```diff
@@ -179,6 +179,9 @@ def collect_fn(self):
             GccBugs.variadic_function(decl, decl_str)
             properties['decl_str'] = decl_str
 
+            if not src_f in self.mod_srcs:
```

Comment:
Is this logic same as the line 169?

Reply:
if not src_f in self.mod_srcs + self.sdcr_srcs:
if not src_f in self.mod_srcs:

They look different, don't they?

Line 169 means, if this file is neither mod_srcs nor sidecar_srcs, we needn't collect syscall functions or declaration string (because these files won't be extracted)

Line 182 means, if this file is not mod_srcs, we needn't collect syscall functions. 

I think this piece can be improved

Reply:
OK, I understand.